common-arcana.lic

```diff
@@ -92,22 +92,31 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = false, runestone_tm = false)
```

Comment:
Is `false` the best default value for `runestone_name` (which is generally a string)?

Reply:
Make it `""` or `''`?

Reply:
Ritual spells were the reason for the false for `runestone_name`. Testing another way.

Reply:
Actually, several other spells kept going into the section when they should not. 

I tried `runestone_name = 'none'` and then `if runestone_name !='none'` but some spells still went into this section. The only setting that works 100% of the time is `runestone_name = false` in the def line.